src/views/LoginFrame.java

```diff
@@ -133,21 +123,19 @@ private void LoginButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-F
         if (LabarotoryJournal.con.checkUserAuth(username, password)) {
             if (LabarotoryJournal.con.checkUserAdmin(username)) {
```

Comment:
Крч втикай
Оцей `LabarotoryJournal.con.checkUserAdmin(username)` рекомендую зробити так:
```suggestion
            isAdmin = LabarotoryJournal.con.checkUserAdmin(username)
```